2024-03-07 02:15:03 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002805575A150>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 11001] getaddrinfo failed)")': /api/4504800232407040/envelope/
2024-03-07 02:15:03 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002805575ABA0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 11001] getaddrinfo failed)")': /api/4504800232407040/envelope/
2024-03-07 02:15:03 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x000002805575BC50>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 11001] getaddrinfo failed)")': /api/4504800232407040/envelope/
2024-03-07 02:15:03 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x0000028055E55D30>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 11001] getaddrinfo failed)")': /api/4504800232407040/envelope/
2024-03-07 02:15:03 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x0000028055E55CD0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 11001] getaddrinfo failed)")': /api/4504800232407040/envelope/
2024-03-07 02:15:03 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x0000028055E56150>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 11001] getaddrinfo failed)")': /api/4504800232407040/envelope/
